preview-maker.js

```diff
@@ -68,9 +71,28 @@ const fill = (node, color) => {
         }
     }
 };
+function rgbToHsl(rgbArr){
+    const hsl = snejugalColor(rgbArr)
+    const result = {hue: hsl.hue, saturation: hsl.saturation * 100, lightness: hsl.lightness * 100, alpha: hsl.alpha}
+    return result;
+}
+const fillHSL = (node, hsl) => {
```

Comment:
Though this function is named `fillHSL`, it actually takes any string and is also a subset of `fill`. So it's better to rename `fill` to `fillRgb`, `fillHSL` to `fill`, make `fillRgb` use `fill` and make another functioin, `fillHsl`:

```js
const fillHsl = (node, hsl) => {
  const hsla = `hsl(${hsl.hue}, ${hsl.saturation * 100}, ${hsl.lightness * 100}, ${hsl.alpha / 255})`;
  fill(node, hsla);
};
```

Note that this function also multiplies everything, so you don't need the `rgbToHsl` function above, you can just import `rgbToHsl` from `@snejugal/color` directly.